content/home.ts

```diff
@@ -31,28 +39,32 @@ const content: HomeContent = {
       items: [
         {
           id: 'how-it-works-item1',
-          title: 'Inform',
           body: 'We break down the laws into understandable language.',
           imgsrc: '/illustrations/teamwork.svg',
         },
         {
           id: 'how-it-works-item2',
-          title: 'Assess',
           body: 'You answer a few simple yes/no questions.',
           imgsrc: '/illustrations/checklist.svg',
         },
         {
           id: 'how-it-works-item3',
-          title: 'Calculate',
           body: 'This helps determine your vacation eligibility in Washington.',
           imgsrc: '/illustrations/teamwork-2.svg',
         },
+        {
+          id: 'how-it-works-item4',
+          body: 'We help you explore what comes after determining your vacation eligibility.',
+          imgsrc: '/illustrations/chart.svg',
+        },
       ],
     },
     {
       id: 'why-vacate',
       title: 'Why Vacate?',
-      subtitle: 'A conviction vacation seals the offense from your record, and will give you more chance to access resources like:',
+      ctaLink: '/get-started',
+      ctaText: 'Get started',
+      subtitle: 'A conviction vacation seals the offense from your record, and will give you more chance to access:',
```

Comment:
Can you please update the subtitle to: "A conviction vacation seals the offense from your record, giving you a better chance at access to:". I made a comment in the Figma file to have it changed there as well. It doesn't sound right to me.

Reply:
updated